examples/object_detection/android-jetpack-compose/app/src/main/java/com/example/objectdetection/composables/MediaPipeBanner.kt

```diff
@@ -0,0 +1,73 @@
+package com.example.objectdetection.composables
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material.icons.filled.Settings
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.example.objectdetection.R
+import com.example.objectdetection.ui.theme.Turquoise
+
+// The MediaPipe banner displayed at the top of the app screens.
+
+// For our purposes, it can show a back button on the left side
+// and an options button on the right side when the corresponding
+// callback functions are provided to it.
+
+// It's intended to provide buttons to navigate between Home and Options screens
+
+@Composable
+fun MediaPipeBanner(
+    onOptionsButtonClick: (() -> Unit)? = null,
+    onBackButtonClick: (() -> Unit)? = null,
+) {
+    Box(
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(56.dp)
+            .background(Color(0xEEEEEEEE)),
+    ) {
+        if (onBackButtonClick != null) {
+            IconButton(
+                onClick = onBackButtonClick,
+                modifier = Modifier.align(Alignment.CenterStart)
+            ) {
+                Icon(
+                    Icons.Filled.ArrowBack,
+                    contentDescription = null,
+                    tint = Turquoise
+                )
+            }
+        }
+        Image(
+            painter = painterResource(id = R.drawable.media_pipe_banner),
+            contentDescription = null,
```

Comment:
Content descriptions are worth having; they're for accessibility and screen readers.

Reply:
Done